Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -70,13 +70,13 @@ def audit
     end
 
     Homebrew.auditing = true
-    inject_dump_stats!(FormulaAuditor, /^audit_/) if args.audit_debug?
+    inject_dump_stats!(FormulaAuditor, /^audit_/) if @args.audit_debug?
```

Comment:
As we're in the `Homebrew` module: can this use either just `args` or `Homebrew.args`?

Reply:
Addressed. Using `args` now